src/App.tsx

```diff
@@ -2,6 +2,7 @@ import { useEffect } from 'react';
 import { Tldraw, useFileSystem } from '@tldraw/tldraw';
 import { useUsers } from 'y-presence';
 import { useCookies } from 'react-cookie';
+
```

Comment:
The correct order we want in all projects between relative pathes and simlinks+packages. Also sorting. But the most do not want the empty line between the imports. If the IDE make the sorting than this line is removed and produce a change line.

Reply:
fixed